### ISSUE TITLE
Thank you very much for your work, I had some problems with visualization

### ISSUE THREAD

**Issue by zhangyu2234** [InhwanBae/ETH-UCY-Trajectory-Visualizer#4]
Hello, when I execute all_pred_traj.append(V_absl[best_traj_idx].permute(1, 0, 2).detach().cpu().numpy()), an error message appears, saying number of dimensions in the tensor input does not match the length of the desired ordering of dimensions i.e. input.dim() = 4 is not equal to len(dims) = 3. This is because V_abs is a four-dimensional tensor. How should I solve this problem?

**Comment by InhwanBae** [comment]
Hi @zhangyu2234,

Thank you for letting me know. I found and fixed a bug in the permutation. It should work properly now.

https://github.com/InhwanBae/ETH-UCY-Trajectory-Visualizer/issues/1#issuecomment-2553027374
